.env.template

```
@@ -0,0 +1,6 @@
+MYSQL_DATABASE=csdl
+MYSQL_USER=csdl
+MYSQL_PASSWORD=csdl
+MYSQL_ROOT_PASSWORD=root
+
+DATABASE_URL=mysql+mysqldb://csdl:csdl@db:3306/csdl
```

Missing blank line at the end
```suggestion
DATABASE_URL=mysql+mysqldb://csdl:csdl@db:3306/csdl

```